packages/owl-bot/src/owl-bot.ts

```diff
@@ -49,11 +50,23 @@ interface PubSubContext {
   };
 }
 
-export function OwlBot(
-  privateKey: string | undefined,
-  app: Probot,
-  db?: Db
-): void {
+async function acquireLock(target: string): Promise<DatastoreLock | boolean> {
+  const lock = new DatastoreLock('owlbot', target, 25 * 1000);
+  if (await lock.peek()) {
+    return false;
+  }
+  const result = await lock.acquire();
```

Comment:
this is taken from library docs;

```js
const result = await lock.acquire();
if (!result) {
	// failure
}
```

I assume the lock returns `false` if it can't be acquired.